dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -233,7 +233,7 @@ public BaseTraceService() {
         systemOut = new SystemLogHolder(LoggingConstants.SYSTEM_OUT, System.out);
         systemErr = new SystemLogHolder(LoggingConstants.SYSTEM_ERR, System.err);
 
-        earlyMessageTraceKiller_Timer.schedule(new EarlyMessageTraceCleaner(), 5 * MINUTE); // 5 minutes wait time
+        earlyMessageTraceKiller_Timer.schedule(new EarlyMessageTraceCleaner(), 1 * MINUTE); // 5 minutes wait time
```

Comment:
@AidanPolese , is change to 1 min temporary?  I'm guessing that's just to make your testing easier somehow.